scripts/phpthemis_postinstall.sh

```diff
@@ -14,6 +14,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -eu
```

Comment:
Below we use pipes, so let's exit with an error code if even one command fails in the chain. 
```suggestion
set -euo pipefail
```